specification/deviceregistry/DeviceRegistry.Management/tspconfig.yaml

```diff
@@ -18,6 +18,7 @@ options:
     flavor: "azure"
     namespace: "com.azure.resourcemanager.deviceregistry"
```

Comment:
this line may need update as well, e.g. https://github.com/Azure/azure-rest-api-specs/pull/31752/files,
please ask weidong if you are not sure how to add configuration

Reply:
Yes, please see if you can migrate to clientNamespace

As I understand, this release will be after a week or two.

Reply:
Fixed by the [commit#1e8614f](https://github.com/Azure/azure-rest-api-specs/pull/31805/commits/1e8614f709a7627ac3daa6f0ca55ac873747f06a)

Reply:
delete this line `namespace`

Reply:
Fixed in the [commit#58b279b](https://github.com/Azure/azure-rest-api-specs/pull/31805/commits/58b279b0a5e2387b78a712b5de53caf8053cb015)